tests/test_api_urls.py

```diff
@@ -42,11 +48,17 @@ def test_api_issues_search(self):
     def test_api_issues_out_of_range(self):
         '''API issue for a non existent number returns JSON 404.'''
         # If we reach 1,000,000 webcompat issues we can celebrate
-        rv = self.app.get('/api/issues/1000000', environ_base=headers)
-        json_body = json.loads(rv.data)
-        self.assertEqual(rv.status_code, 404)
-        self.assertEqual(rv.content_type, 'application/json')
-        self.assertEqual(json_body['status'], 404)
+        with patch('requests.get') as github_data:
+            api_response = MagicMock(spec=Response)
+            api_response.status_code = 404
+            api_response.content_type = 'application/json'
+            api_response.data = '{"message":"Not Found. Lost in Punk Cat Space","status":404}'  # nopep8
```

Comment:
Cool, that makes a lot of sense. I'll also change the patch method then, to patch not 'requests.get', but to patch 'webcompat.helpers.proxy_request' which is where we actually make the GithubAPI call. If I leave it like it is now, it's mocking all get requests, even to our own pages.